ros2_control_demo_robot/controllers/rrbot_gazebo_forward_controller_position.yaml

```diff
@@ -0,0 +1,16 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 100  # Hz
+
+    joint_trajectory_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+
+    joint_state_controller:
+      type: joint_state_controller/JointStateController
+
+joint_trajectory_controller:
+  ros__parameters:
+    joints:
+      - joint1
+      - joint2
+    interface_name: position
```

Comment:
@destogl can you explain what that interface name does here? I'd assume that the joints are sufficiently described in the ros2_control tag with their control and state interfaces names.
What if I use `position` here and declare the joints as `effort`?